sdk/appconfiguration/azure-appconfiguration/samples/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-app-configuration
+---
+
+# Azure App Configuration Data Library Python Samples
+
+## Prerequisites
+
+You must have an [Azure subscription][azure_sub], and a [Configuration Store][configuration_store] to use this package.
+
+To create a Configuration Store, you can use the Azure Portal or [Azure CLI][azure_cli].
+
+After that, create the Configuration Store:
+
+```Powershell
+az appconfig create --name <config-store-name> --resource-group <resource-group-name> --location eastus
+```
+
+## Setup
+
+Install the Azure App Configuration client library for Python with pip:
+
+```commandline
+pip install azure-appconfiguration
+```
+
```

Comment:
is there no dependency on identity?

Reply:
No. azure-core does not depend on identity. We will have identity dependencies after we have AAD support.

Reply:
I meant in the samples :)

Reply:
Right. Samples do not depend on identity either. :)